bitnami/cilium/values.yaml

```diff
@@ -2069,6 +1953,1437 @@ envoy:
+        ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+        ## e.g:
+        ## interval: 10s
+        ##
+        interval: ""
+        ## @param hubble.relay.metrics.serviceMonitor.scrapeTimeout Timeout after which the scrape is ended
+        ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
+        ## e.g:
+        ## scrapeTimeout: 10s
+        ##
+        scrapeTimeout: ""
+        ## @param hubble.relay.metrics.serviceMonitor.metricRelabelings Specify additional relabeling of metrics
+        ##
+        metricRelabelings: []
+        ## @param hubble.relay.metrics.serviceMonitor.relabelings Specify general relabeling
+        ##
+        relabelings: []
+        ## @param hubble.relay.metrics.serviceMonitor.selector Prometheus instance selector labels
+        ## ref: https://github.com/bitnami/charts/tree/main/bitnami/prometheus-operator#prometheus-configuration
+        ## selector:
+        ##   prometheus: my-prometheus
+        ##
+        selector: {}
+
+  ## @section Hubble UI Parameters
+  ##
+  ui:
+    ## @param hubble.ui.enabled Enable Hubble UI
+    ##
+    enabled: false
+    ## Frontend specific parameters
+    ##
+    frontend:
+      ## Bitnami Hubble UI image
+      ## ref: https://hub.docker.com/r/bitnami/hubble-ui/tags/
+      ## @param hubble.ui.frontend.image.registry [default: REGISTRY_NAME] Hubble UI image registry
+      ## @param hubble.ui.frontend.image.repository [default: REPOSITORY_NAME/hubble-ui] Hubble UI image repository
+      ## @skip hubble.ui.frontend.image.tag Hubble UI image tag (immutable tags are recommended)
+      ## @param hubble.ui.frontend.image.digest Hubble UI image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag image tag (immutable tags are recommended)
+      ## @param hubble.ui.frontend.image.pullPolicy Hubble UI image pull policy
+      ## @param hubble.ui.frontend.image.pullSecrets Hubble UI image pull secrets
+      ## @param hubble.ui.frontend.image.debug Enable Hubble UI image debug mode
+      ##
+      image:
+        registry: docker.io
+        repository: bitnami/hubble-ui
+        tag: 0.13.1-debian-12-r0
+        digest: ""
+        ## Specify a imagePullPolicy
+        ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+        ## ref: https://kubernetes.io/docs/concepts/containers/images/#pre-pulled-images
+        ##
+        pullPolicy: IfNotPresent
+        ## Optionally specify an array of imagePullSecrets.
+        ## Secrets must be manually created in the namespace.
+        ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+        ## e.g:
+        ## pullSecrets:
+        ##   - myRegistryKeySecretName
+        ##
+        pullSecrets: []
+        debug: false
```

Comment:
Value not used

Reply:
addressed at https://github.com/bitnami/charts/pull/27526/commits/6e1f6e34bf685be14e71fc180dee89c349f180ab